src/styles/getModalStyles.js

```diff
@@ -4,7 +4,7 @@ import variables from './variables';
 import themeColors from './themes/default';
 
 export default (type, windowDimensions, popoverAnchorPosition = {}) => {
-    const {isSmallScreenWidth, windowWidth} = windowDimensions;
+    const {isSmallScreenWidth, windowWidth, isLandscape} = windowDimensions;
```

Comment:
It seems premature to add `isLandscape`. We are only using this in one place and using `withWindowDimensions` in a zillion places. Could we alternatively just derive this by looking at the `windowWidth` and `windowHeight`?

Reply:
Okay sure. Do you see that in near future we won't be using it often? If yes, then I'll add the check here itself.

Reply:
Updated

Reply:
That's the million dollar question. Our engineering culture at Expensify usually says **wait until we need the code** not **create with the expectation that something will be needed in the future**. Most of the time this attitude leads to less unnecessary complexity - and only sometimes leads to repetition. But it's easier to identify repetition than it is to identify things that are only used once.

Reply:
Noted. Thanks for clarifying. Helpful for making better contributions 🙇